invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/administration/records/search/SearchResultItemLayout.js

```diff
@@ -64,9 +64,9 @@ class SearchResultItemComponent extends Component {
             color="red"
             icon="lock"
           />
-          <a target="_blank" rel="noreferrer" href={result.links.self_html}>
+          <a target="_blank" rel="noreferrer noopener" href={result.links.self_html}>
             {_truncate(result.metadata.title || i18next.t("Empty draft title"), {
-              length: 50,
+              length: 100,
```

Comment:
we can replace the truncate with CSS solution https://github.com/inveniosoftware/invenio-app-rdm/commit/c97c82825e5c178cdee623a0c70bb357a641a846 (you would need a new class `truncate-lines-1`
the problem there was that truncate function disregards closing tags completely - for example when used with latex, which causes the formulas to display in a broken way. CSS handles that case way better (tested on many browsers already)

Reply:
If we want to truncate it we can't have it being inline as it [displays as a -webkit-box](https://github.com/carlinmack/invenio-theme/blob/remove-unused/invenio_theme/assets/semantic-ui/less/invenio_theme/theme/globals/site.overrides#L419)

![image](https://github.com/user-attachments/assets/40577dc0-203a-4264-a1fc-863ed7ac7d64)

(using `truncated` instead of `truncate-lines-1` does not work as there is no max width as shown)

![image](https://github.com/user-attachments/assets/9f0a7bd5-0f4d-4d15-85a7-58eb456d384b)

It is weird and unintuitive. Maybe this is why `truncate` in Jinja was chosen

Reply:
I think longterm, using CSS might be the better option, since:

- there's no information loss
- no JS necessary
- better handling for "split" formulas/phrases/markup
- weirdly also works when searching for text that is truncated (even though you don't see the highlighted result, but I maybe there's a trick for that)